tests/test_network.py

```diff
@@ -1247,6 +1247,40 @@ def test_get_empty_attributes(self):
         self.assertEquals(gens.index.tolist(), ['GEN', 'GEN2'])
         self.assertTrue(gens.columns.empty)
 
+    def test_properties(self):
+        pp.set_debug_mode(True)
+        network = pp.network.create_eurostag_tutorial_example1_network()
+        properties = pd.DataFrame.from_records(index='id', data=[
+            {'id': 'GEN', 'prop1': 'test_prop1', 'prop2': 'test_prop2'},
+            {'id': 'NHV1_NHV2_1', 'prop1': 'test_prop1', 'prop2': 'test_prop2'}
+        ])
+        network.add_elements_properties(properties)
+        expected = pd.DataFrame.from_records(index='id',
+                                             data=[{'id': 'NHV1_NHV2_1', 'prop2': 'test_prop2', 'prop1': 'test_prop1'},
+                                                   {'id': 'NHV1_NHV2_2', 'prop2': '', 'prop1': ''}])
+        pd.testing.assert_frame_equal(network.get_lines(attributes=['prop1', 'prop2']), expected, check_dtype=False)
+        expected = pd.DataFrame.from_records(index='id',
+                                             data=[{'id': 'GEN', 'prop2': 'test_prop2', 'prop1': 'test_prop1'},
+                                                   {'id': 'GEN2', 'prop2': '', 'prop1': ''}])
+        pd.testing.assert_frame_equal(network.get_generators(attributes=['prop1', 'prop2']), expected, check_dtype=False)
+        network.add_elements_properties(id='NHV1_NHV2_2', prop3='test_prop3')
+        expected = pd.DataFrame.from_records(index='id',
+                                             data=[{'id': 'NHV1_NHV2_1', 'prop2': 'test_prop2', 'prop1': 'test_prop1', 'prop3': ''},
+                                                   {'id': 'NHV1_NHV2_2', 'prop2': '', 'prop1': '', 'prop3': 'test_prop3'}])
+        pd.testing.assert_frame_equal(network.get_lines(attributes=['prop1', 'prop2', 'prop3']), expected, check_dtype=False)
+        removed_properties = pd.DataFrame.from_records(index='id', data=[
+            {'id': 'GEN', 'properties': 'prop1'},
+            {'id': 'GEN', 'properties': 'prop2'}
+        ])
+        network.remove_elements_properties(removed_properties)
+        columns = network.get_generators(all_attributes=True).columns
+        self.assertTrue('prop1' not in columns and 'prop2' not in columns and 'prop3' not in columns)
+        network.remove_elements_properties(id=['NHV1_NHV2_2'], properties=['prop3'])
+        network.remove_elements_properties(id=['NHV1_NHV2_1', 'NHV1_NHV2_1'], properties=['prop2', 'prop1'])
```

Comment:
Actually the behaviour I expected was that
```python
network.remove_elements_properties(id='NHV1_NHV2_1', properties=['prop2', 'prop1'])
```
would remove both properties on the line NHV1_NHV2_1.

And
```python
network.remove_elements_properties(id=['E1', 'E2', 'E3'], properties=['prop2', 'prop1'])
```
would remove prop2 and prop1 for the 3 elements E1, E2, and E3.